docs/How/integration_checklist.md

```diff
@@ -0,0 +1,102 @@
+# Steps for a New Integration
+
+## Overview
+
+Below, you will find a checklist of the steps that should be followed by any project maintainers when performing a new
+upstream integration. This is for reference by the maintainers and by any of the community interested in the process.
+
+We will do our best to keep this up to date, so it's an excellent reference for what to expect if you're waiting on an
+integration.
+
+## 1) Announce
+
+Advertise the upcoming update, where it will start from, how long it will take, and what will happen with the existing
+release branch afterwards. This should be done in the `Release Announcements` channel in the Project Mu Team.
+
+## 2) Rebase and Test
+
+Much of this process is designed to be run on every Mu repo. When deciding which repos to update first, refer to the CI
+dependencies documented in each repo.
```

Comment:
A link to given one of the CI dependency sections in a repo to serve as an example for the others would be helpful.